src/pages/publication/PublicationFormTabs.tsx

```diff
@@ -39,26 +40,29 @@ export const PublicationFormTabs: FC<PublicationFormTabsProps> = ({ handleTabCha
   return (
     <Tabs variant="fullWidth" value={tabNumber} onChange={handleTabChange} aria-label="navigation" textColor="primary">
       <LinkTab
-        label={`1. ${t('heading.description')}`}
+        label={`1. ${isMobile ? '' : t('heading.description')}`}
```

Comment:
lurer på om vi egentlig ha TS-logikk for å bestemme dette. hvis man feks bytter mellom landscape og portrait vil det jo bli kluss. burde det ikke heller ligge i noe viewport-css-greier? evt vil jeg tro at MUI tabs får det til å virke greit om vi setter `scrollButtons="auto"`(?)

Reply:
ups, utdatert commit :)